oteapi/strategies/transformation/celery_remote.py

```diff
@@ -30,19 +29,21 @@
 
 
 class CeleryConfig(AttrDict):
-    """Celery configuration."""
+    """Celery configuration.
 
-    task_name: str = Field(..., description="A task name.")
+    All fields here (including those added from the session through the `get()` method,
+    as well as those added "anonymously") will be used as keyword arguments to the
+    `send_task()` method for the Celery App.
+    """
+
+    name: str = Field(..., description="A task name.")
```

Comment:
Note that this will break the current demonstrators which use task_name in the configuration.

Reply:
I understand. But this is the correct argument name for the `send_task()` method. I could get around it by using `alias`?

Reply:
I have implemented using `alias`, as well as ensuring non-aliases can be used to instantiate the strategy-specific configuration model (`CeleryConfig`). Furthermore, I've expanded the `_use_session()` method to include the possibility of using aliases in the session object.